src/HJSON/HJSONParser.php

```diff
@@ -99,23 +100,28 @@ private function value()
             case '[':
                 return $this->_array();
             case '"':
-                return $this->string();
+                return $this->string('"');
+            case '\'':
+                if ($this->peek(0) !== '\'' || $this->peek(1) !== '\'') {
+                    return $this->string('\'');
+                }
+                // Falls through on multiline strings
             default:
                 return $this->tfnns();
         }
     }
 
-    private function string()
+    private function string($quote)
```

Comment:
The `$quote` will be the `double` or `single`.

Is it necessary to set the double quote for `$quote` variable by default?

Reply:
No, not really.

Reply:
Ok.